jujugui/static/gui/src/app/utils/statsd.js

```diff
@@ -19,24 +19,25 @@ const StatsClient = class StatsClient {
   */
   constructor(url, prefix='', flags={}) {
     this.url = url.replace(/\/?$/, '/');
-    this.flags = flags;
+    this._flags = flags || {};
```

Comment:
So flags default to {} in the declaration, and according to hatch juju_config.flags is always defined. I don't think we need the `|| {}` bit. Not that there's any harm in having it.